light_curves/multiband_lc.ipynb

```diff
@@ -68,35 +78,11 @@
     "import numpy as np\n",
     "from astropy.table import Table\n",
     "from scipy import stats\n",
+    "import pandas as pd\n",
     "\n",
     "warnings.filterwarnings('ignore')\n",
     "\n",
-    "# these don't need to be imported but they do need to be installed\n",
-    "# so leaving the try/excepts for now\n",
-    "try:\n",
-    "    from tqdm import tqdm\n",
-    "except ImportError:\n",
-    "    !pip install tqdm\n",
-    "    from tqdm import tqdm\n",
-    "\n",
-    "try:\n",
-    "    import lightkurve as lk\n",
-    "except ImportError:\n",
-    "    !pip install lightkurve --upgrade\n",
-    "    import lightkurve as lk\n",
-    "        \n",
-    "try:\n",
-    "    from acstools import acszpt\n",
-    "except ImportError:\n",
-    "    !pip install acstools\n",
-    "    from acstools import acszpt\n",
-    "\n",
-    "try:\n",
-    "    from alerce.core import Alerce\n",
-    "except ImportError:\n",
-    "    !pip install alerce\n",
-    "    from alerce.core import Alerce\n",
-    "\n",
+    "#get tools for accessing Meisner et al., 2023 WISE light curves\n",
     "try:\n",
     "    from unTimely_Catalog_tools import unTimelyCatalogExplorer\n",
     "except ImportError:\n",
```

Comment:
I think these installs can go in the requirements file as well, and this whole try/except block can be deleted. (GitHub won't let me make the specific code suggestion since this block isn't changed by this PR.)

Reply:
ok, thanks, pushed as a separate commit